dune-project

```diff
@@ -53,6 +53,7 @@ possible and does not make any assumptions about IO.
   dot-merlin-reader
   (csexp (>= 1.2.3))
   (result (>= 1.5))
+  (reason :with-test)
```

Comment:
Can we drop reason from the tests? I recently removed it because they don't publish it regularly on opam: https://github.com/ocaml/ocaml-lsp/issues/247

If you'd like to exercise reason functionality, I'd suggest to create a mock `refmt` binary. I think it's fine to just drop it though.

Reply:
I'm not sure how I feel about supporting reason, but having no tests for it. If I understand correctly, release on Opam is the issue, but there's nothing blocking us from using reason in development, right?

Reply:
> If I understand correctly, release on Opam is the issue, but there's nothing blocking us from using reason in development, right?

There are issues with setting it up in CI however.

> I'm not sure how I feel about supporting reason, but having no tests for it.

We can add tests for it, but we just need to mock `refmt`. As long as we match the cli, the tests using the mock binary would be just as valuable.

Reply:
Ok, I'll remove the tests from this PR and will bring them back in another one - don't want to delay the merge any longer if that's the only blocker 🙂